src/bridge/EspressoTEEVerifier.sol

```diff
@@ -57,7 +47,7 @@ contract EspressoTEEVerifier is Ownable {
         @param rawQuote The quote from the TEE
         @param reportDataHash The hash of the report data
     */
-    function verify(bytes calldata rawQuote, bytes32 reportDataHash) external {
+    function verify(bytes calldata rawQuote, bytes32 reportDataHash) external view {
```

Comment:
looks good! Since the method does not explicitly provide a success indicator, could be good to explain in the comments that the verification is successful if the function doesn't revert.

Reply:
Sounds good, will do!